geode-redis/src/main/java/org/apache/geode/redis/internal/executor/SingleResultRedisFunction.java

```diff
@@ -64,11 +61,4 @@ public boolean optimizeForWrite() {
   public boolean isHA() {
     return true;
   }
-
-  public static void computeWithPrimaryLocked(Object key, LocalDataSet localDataSet, Runnable r) {
```

Comment:
I presume it's OK that we're not using the `LocalDataSet` anymore? On a related note `RenameFunction.getLocalDatSet` can be removed.

Reply:
Yes. If, in the future, we have an operation that needs the LocalDataSet then we can pass it to that operation explicitly.
I moved RenameFunction.getLocalDataSet